app/kumactl/cmd/config/config_control_planes_add_test.go

```diff
@@ -155,3 +165,22 @@ switched active Control Plane to "example"
 		)
 	})
 })
+
+func setupCpIndexServer() int {
```

Comment:
It would be great if we could re-use actual `HandleFunc` from `pkg/api-server/index_ws.go`

Reply:
so the problem is that the function in gorestful has different signature than in golang builtin http server. I'd rather use `httptest` in tests.